README.md

```diff
@@ -111,13 +120,39 @@ If you would like to publish to your own domain, you will need to specify these
 | route           | The route you would like to publish to                                 | `route = "example.com/my-worker/*"`            |
 | zone_id         | Your Cloudflare zone ID, this can be found in the Cloudflare dashboard | `zone_id = "b6558acaf2b4cad1f2b51c5236a6b972"` |
 
+=======
+
+  If you want to publish to [workers.dev](https://workers.dev), you will first need to have a [workers.dev](https://workers.dev) subdomain registered. You can register a subdomain by executing:
+
+  ```bash
+  wrangler subdomain <name>
+  ```
+
+  After you have registered a subdomain, add `workers_dev` to your `wrangler.toml`.
+
+| Key             | Value | Example                  |
+| --------------- | ----- | ------------------------ |
+| workers_dev | true  | `workers_dev = true` |
+
+#### Publishing to your own domain
+
+If you would like to publish to your own domain, you will need to specify these three fields in your `wrangler.toml`.
+
+| Key             | Value                                                                  | Example                                        |
+| --------------- | ---------------------------------------------------------------------- | ---------------------------------------------- |
+| workers_dev | false                                                                  | `workers_dev = false`                      |
+| route           | The route you would like to publish to                                 | `route = "example.com/my-worker/*"`            |
+| zone_id         | Your Cloudflare zone ID, this can be found in the Cloudflare dashboard | `zone_id = "b6558acaf2b4cad1f2b51c5236a6b972"` |
+
+>>>>>>> master
 #### Publishing the same code to multiple places
 
 If you would like to be able to publish your code to multiple places, please see the documentation for [environments](https://github.com/cloudflare/wrangler/blob/master/docs/environments.md).
```

Comment:
all instances of docs/environments.md need to be changed to docs/content/environments.md